runner/src/server/plugins/engine/types.ts

```diff
@@ -43,6 +45,7 @@
 export type FormSubmissionState = {
   progress?: string[];
   [propName: string]: any;
+  callback?: InitialiseSessionOptions;
```

Comment:
you didnt update the jsdoc

Reply:
changes pulled in from mai like the others